starknet_py/contract.py

```diff
@@ -120,13 +120,14 @@ class DeclareResult(SentTransaction):
     compiled_contract: str = None  # pyright: ignore
 
     def __post_init__(self):
-        if any(
-            field is None
-            for field in [self.class_hash, self._account, self.compiled_contract]
-        ):
-            raise ValueError(
-                "None of the account, class_hash and compiled_contract fields can be None"
-            )
+        if self._account is None:
+            raise ValueError("Argument _account can't be None.")
+
+        if self.class_hash is None:
+            raise ValueError("Argument class_hash can't be None.")
+
+        if self.compiled_contract is None:
+            raise ValueError("Argument compiled_contract can't be None.")
```

Comment:
Could we somehow unify them? Error message is the same, only argument name is different. Any ideas?

Reply:
You could move ValueError creation to a method, but I think that would be overkill.

Reply:
I think we can keep it that way, without doing any shenanigans. At least it's explicit.